roles/remove-node/post-remove/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+delete_node_retries: 10
+delete_node_delay_seconds: 3
```

Comment:
Nitpick, missing new line at end of file. How come the yamllint job did not fail on this?

Reply:
Thanks, I fixed it.